README.md

```diff
@@ -107,7 +107,7 @@ You will get errors from most tools if they have both workspaces in their enviro
 First we start a ROS 1 `roscore`:
 
 ```
-# Shell A:
+# Shell A (ROS1 only):
```

Comment:
Please use a space between `ROS` and the major version number.

Same below multiple times.